exasol_integration_test_docker_environment/testing/docker_registry.py

```diff
@@ -18,7 +18,7 @@ def default_docker_repository_name(env_name: str) -> str:
 
 
 class LocalDockerRegistry:
-    def __init__(self, name: str, registry_container, registry_port):
+    def __init__(self, name: str, registry_container, registry_port) -> None:
```

Comment:
are the missing types intended

Reply:
Yes, will be fixed in https://github.com/exasol/integration-test-docker-environment/issues/453